resources/check.rb

```diff
@@ -24,4 +24,7 @@ def after_created
   unless name =~ /^[\w\.-]+$/
     raise Chef::Exceptions::ValidationFailed, "Sensu check name cannot contain spaces or special characters"
   end
+  
+    raise Chef::Exceptions::ValidationFailed, "Sensu check must either define subscribers, or has to be standalone." unless (subscribers || standalone)
```

Comment:
I do like this validation check though; I think it could save some people some needless debugging.